src/lib/drishti/CMakeLists.txt

```diff
@@ -230,7 +230,9 @@ endif(DRISHTI_BUILD_WORLD)
 # iOS + QT faces a similar issues, which may be more easily resolved. Android emulator
 # may be another option for providing an OpenGL context in native code.
 # The near term intention is to support this: if(DRISHTI_BUILD_QT OR (NOT ${DRISHTI_IS_MOBILE}))
-if(NOT ${DRISHTI_IS_MOBILE} AND NOT ${drisht_ci})
+
+#if(NOT ${DRISHTI_IS_MOBILE} AND NOT ${drisht_ci})
+if(NOT ${DRISHTI_CI})
```

Comment:
Testing should work for all locally hosted scenarios (mobile or desktop).  CI is next...